c/openbsd-etherip/README

```diff
@@ -0,0 +1,8 @@
+Contributed by: Moritz Buhl
+
+Taken from the OpenBSD 6.2 src checkout https://www.openbsd.org/
+Copyright and license are mentioned in each file of the source directory.
+
+The create.sh file shows how to create the if_etherip_*.i files.
```

Comment:
The file `create.sh` doesn't exist anymore, could you adjust this?

Reply:
Sorry, I should have rechecked.